pkg/apiserver/api/v1/request.proto

```diff
@@ -29,3 +29,10 @@ message ListRequest {
   int64 limit = 3;
   string continue = 4;
 }
+
+message WatchRequest {
+  string account = 1;
+  string labelSelector = 2;
+  string resourceVersion = 3;
+  string operationType = 4;
```

Comment:
Do we even need this filter by operation type - Kubernetes also does not support such a feature and I can't think of a use case for it.

Reply:
done